src/diffusers/pipelines/wuerstchen/pipeline_wuerstchen_combined.py

```diff
@@ -0,0 +1,250 @@
+# Copyright 2023 The HuggingFace Team. All rights reserved.
```

Comment:
Can we add tests for this class/file as is done here: https://github.com/huggingface/diffusers/blob/main/tests/pipelines/kandinsky_v22/test_kandinsky_combined.py

Reply:
You are doing this right?